mpdel-tablist.el

```diff
@@ -108,7 +108,8 @@
 
 (navigel-method mpdel navigel-entity-to-columns ((song libmpdel-song))
   (vector
-   (propertize (or (libmpdel-entity-name song) "") 'face 'mpdel-tablist-song-name-face)
+   (propertize (or (libmpdel-entity-name song)
+                   (libmpdel-song-file song) "") 'face 'mpdel-tablist-song-name-face)
```

Comment:
```suggestion
   (propertize (or (libmpdel-entity-name song)
                   (libmpdel-song-file song) 
                   "") 'face 'mpdel-tablist-song-name-face)
```

This may be a nitpick available with time, but it may be better to move the default, empty string (`""`) onto its own line.